Source/WebCore/css/CSSToLengthConversionData.h

```diff
@@ -65,15 +65,16 @@ class CSSToLengthConversionData {
     const RenderView* renderView() const { return m_renderView; }
     const Element* elementForContainerUnitResolution() const { return m_elementForContainerUnitResolution.get(); }
 
+    const RenderStyle* styleForFontUnits() const { return computingFontSize() ? parentStyle() : style(); }
```

Comment:
I think this could be narrowed to fontCascadeForFontUnits/computedLineHeightForFontUnits. Nothing else should be coming from the parent style.